packages/tools/locales/fr_FR.po

```diff
@@ -718,7 +718,7 @@ msgstr "Déchiffrement complété."
 
 #: packages/lib/Synchronizer.ts:196
 msgid "Completed: %s (%s)"
-msgstr "Terminé : %s (%s)"
+msgstr "Fait le : %s (%s)"
```

Comment:
I don't think there's much improvement there since it's the same length, so please revert to how it was.

Reply:
I agree.  imho the "Completed" label can be fully removed from there. It's seems useless as there is enough context already (with the status details visible just before and the "Syncrhonise" button visible  just after).

Reply:
I'm 100% sure [we'll break someone's workflow](https://xkcd.com/1172/) if we remove this label so I prefer we leave it for now. So let's leave the translation as it was too then.